packages/web/components/Quest/QuestForm.tsx

```diff
@@ -87,7 +90,15 @@ const getDefaultFormValues = (
       label: s.name,
       ...s,
     })),
-  roles: [],
+  // fix this
```

Comment:
how? 😁

Reply:
i have this fixed in the client PR, i just added that cause of eslint complaining :)

Reply:
oh wait, this is the client pr. x) just need to clear the comment